python/setup.py

```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/python
+
+# Python imports
+import os
+from setuptools import setup
+
+setup(
+    name="aruba-sensor-monitoring",
+    version="1.0.0",
+    author="Antti-Pekka Meronen",
+    author_email="antti-pekka.meronen@digia.com",
+    description="Monitoring scripts for Aruba sensors",
+    url="https://github.com/digiaiiris/zabbix-aruba-sensor-monitoring/",
+    license="GPLv3",
```

Comment:
Suggesting MIT

Reply:
Switched in commit 938ead6c1f8b13f9d884da2a8c521baf387d6278.